### ISSUE TITLE
aiohttp requirement missing from setup.py

### ISSUE THREAD

**Issue by carlwgeorge** [aio-libs/aiodocker#8]
This library [imports aiohttp](https://github.com/paultag/aiodocker/blob/285c4ee7a409b4c941fbbbc0795e4e01d28a1d3f/aiodocker/docker.py#L5), but doesn't specify it as a requirement in setup.py.  This results in errors such as the following.

```
(aiodocker) $ pip install aiodocker
Collecting aiodocker
  Downloading aiodocker-0.6.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-wwzirc4b/aiodocker/setup.py", line 4, in <module>
        from aiodocker import __version__
      File "/tmp/pip-build-wwzirc4b/aiodocker/aiodocker/__init__.py", line 3, in <module>
        from .docker import Docker
      File "/tmp/pip-build-wwzirc4b/aiodocker/aiodocker/docker.py", line 5, in <module>
        import aiohttp
    ImportError: No module named 'aiohttp'

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-wwzirc4b/aiodocker/
```

**Comment by carlwgeorge** [comment]
@paultag ping?

**Comment by trivigy** [comment]
Hey @carlwgeorge note that the dependency is inside the requirements file. Just run `pip install -r /path/to/requirements.txt` in the meantime. Hopefully that will change soon.

**Comment by achimnol** [comment]
This is now fully fixed in v0.7 as we updated aiodocker to work with aiohttp v2.
